src/auto/text_buffer.rs

```diff
@@ -274,9 +284,13 @@ impl TextBuffer {
         }
     }
 
-    //pub fn get_serialize_formats(&self) -> (/*Ignored*/Vec<gdk::Atom>, i32) {
-    //    unsafe { TODO: call ffi::gtk_text_buffer_get_serialize_formats() }
-    //}
+    pub fn get_serialize_formats(&self) -> (Vec<gdk::Atom>, i32) {
```

Comment:
This function need ignored or implemented manually.
Test shows 1/6 and fails on printing second name.
```
let buffer = gtk::TextBuffer::new(None);
buffer.set_text("test");

let (formats, count) = buffer.get_serialize_formats();
println!("{}/{}", count, formats.len());
for format in formats {
    println!("{}", format.name());
}
```